tar/subst.c

```diff
@@ -128,9 +128,14 @@ add_substitution(struct bsdtar *bsdtar, const char *rule_text)
 	rule->regular = 1; /* Rewrite regular filenames. */
 	rule->symlink = 1; /* Rewrite symlink targets. */
 	rule->hardlink = 1; /* Rewrite hardlink targets. */
+	rule->from_begin = 0; /* Don't match from start. */
 
 	while (*++end_pattern) {
 		switch (*end_pattern) {
+		case 'b':
+		case 'B':
```

Comment:
This was done, because g is handled in the same way, even though it feels a bit wasteful.